ci_framework/playbooks/ceph.yml

```diff
@@ -180,6 +180,14 @@
         name: cifmw_cephadm
         tasks_from: pools
 
+    - name: Deploy additional daemons
+      ansible.builtin.import_role:
+        name: cifmw_cephadm
+        tasks_from: rgw
+      vars:
+        cifmw_cephadm_rgw_network: 192.168.122.0/24
+        cifmw_cephadm_rgw_vip: 192.168.122.22/24
```

Comment:
Since we could use any VIP here, we could have a pre_task set this too. 

We know the range (as described above) and we know which IPs in that range are in use on the host from `host_to_ip` so you could just take any IP which is not in `host_to_ip`

https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/playbooks/ceph.yml#L93-L98

Reply:
Right, a `pre_task` where we compute the `VIP` that will be used for `RGW` and `ceph-nfs` is a good idea. We can reserve it in advance (filtering the already assigned IPs in the Storage network) and propagate it to the cephadm role.